codegen/module.go

```diff
@@ -1212,15 +1336,16 @@ func FormatGoFile(filePath string) error {
 }
 
 // ModuleClass defines a module class in the build configuration directory.
-// THis could be something like an Endpoint class which contains multiple
+// This could be something like an Endpoint class which contains multiple
 // endpoint configurations, or a Lib class, that is itself a module instance
 type ModuleClass struct {
-	Name        string
-	ClassType   moduleClassType
-	Directories []string
-	DependsOn   []string
-	DependedBy  []string
-	types       map[string]BuildGenerator
+	Name           string
+	ClassType      moduleClassType
+	Directories    []string
+	DependsOn      []string
+	DependedBy     []string
+	DefaultDepDirs []string // Default Dependencies
```

Comment:
If you put the comment on top of the line, IDEs will recognize it's a comment for the field and displays it when proper.

Reply:
either way, the comment doesn't appear to be adding much here.